packages/examples/src/rule.ts

```diff
@@ -41,7 +41,7 @@ export const schema = {
 };
 
 export const uischema = {
-  type: 'HorizontalLayout',
+  type: 'VerticalLayout',
```

Comment:
I'd rather a new, explicit example for demonstrating the radio button group and revert this change.